api/v1alpha1/serviceclaim_types.go

```diff
@@ -60,12 +60,16 @@ type ServiceClaimApplicationClusterContext struct {
 
 // ServiceClaimStatus defines the observed state of ServiceClaim
 type ServiceClaimStatus struct {
+	// The state of the ServiceClaim observed
 	//+kubebuilder:validation:Enum=Pending;Resolved;Invalid
 	//+kubebuilder:default:=Pending
-	State             ServiceClaimState  `json:"state"`
-	ClaimID           string             `json:"claimID,omitempty"`
-	RegisteredService string             `json:"registeredService"`
-	Conditions        []metav1.Condition `json:"conditions,omitempty"`
+	State ServiceClaimState `json:"state"`
+	// Unique ID For the ServiceClaim
+	ClaimID string `json:"claimID,omitempty"`
+	// Name of the RegisteredService the ServiceClaim claimed
```

Comment:
Can be changed to `Name of RegisteredService Claimed`

Reply:
done